src/org/jitsi/impl/neomedia/rtp/TransportCCEngine.java

```diff
@@ -320,60 +384,64 @@ public void onReceiveBitrateChanged(Collection<Long> ssrcs, long bitrate)
     @Override
     public void tccReceived(RTCPTCCPacket tccPacket)
     {
-        MediaStream videoStream = null;
-        for (MediaStream stream : mediaStreams)
+        RTCPTCCPacket.PacketMap packetMap = tccPacket.getPackets();
+        long previousArrivalTimeMs = -1;
+        for (Map.Entry<Integer, Long> entry : packetMap.entrySet())
         {
-            if (stream instanceof VideoMediaStream)
+            long arrivalTime250Us = entry.getValue();
```

Comment:
i'm confused by this name now :) what does this represent?

Reply:
The time unit that is used for the deltas in the TCC packet is 250us. I wonder if there's a better name that I could use.

Reply:
gotcha, so if this had a value of 1, that represents 250us...if it was 2, it would be 500us, yeah? if that's the case then the name doesn't seem bad to me.

Reply:
> gotcha, so if this had a value of 1, that represents 250us...if it was 2, it would be 500us, yeah?

right